.github/workflows/rust.yml

```diff
@@ -39,13 +40,23 @@ jobs:
           available_crates="$(find . -name Cargo.toml -printf '%h\n' | sed 's:./::' | sort | tac)"
           changed_files="$(git diff --name-only HEAD^ HEAD)"
 
-          changed_crates=""
+          # All jobs for all crates will run if any of those paths changes
+          always_run_on=(".github" "rust-toolchain.toml")
+
+          # We ensure, that `miri` will run for these crates
```

Comment:
```suggestion
          # Ensure that `miri` runs for these crates
```

Reply:
Why are we doing this if the crates aren't changed?

Reply:
The comment is misleading, it will only run those crates if they changed. Will be obsolete with #558

Reply:
Gotcha, is there a reason we can't just do these in the same PR?

Reply:
(I was about to ask why we don't just run miri when any crate changes)

Reply:
> Gotcha, is there a reason we can't just do these in the same PR?

This PR was already opened and I actually didn't expect #558 to be that small.

Reply:
> (I was about to ask why we don't just run miri when any crate changes)

This would require disabling almost every test in `packages/engine` if miri runs, which isn't great I think